libs/designer/src/lib/ui/panel/templatePanel/createWorkflowPanel/usePanelTabs.tsx

```diff
@@ -62,30 +65,56 @@ export const useCreateWorkflowPanelTabs = ({
     }
   }, [dispatch, mapping, existingWorkflowName, existingWorkflowNames, parametersExist, selectedTabId, kind]);
 
-  const handleCreateClick = useCallback(async () => {
-    setIsLoadingCreate(true);
-    await onCreateClick(() => setIsCreated(true));
-    setIsLoadingCreate(false);
-  }, [onCreateClick]);
+  const { isLoading: isCreating, mutate: createWorkflowFromTemplate } = useMutation(async () => {
+    setErrorMessage(undefined);
+    const logId = LoggerService().startTrace({
+      name: 'Create Workflow from Template',
+      action: 'createWorkflowFromTemplate',
+      source: 'CreateTabs',
+    });
+    try {
+      await onCreateClick();
+      setIsCreated(true);
+      LoggerService().log({
+        level: LogEntryLevel.Trace,
+        area: 'Templates.usePanelTabs',
+        message: 'Template is created',
+        args: [templateName, workflowAppName],
+      });
+      LoggerService().endTrace(logId, { status: Status.Success });
+    } catch (e: any) {
+      setErrorMessage(e.message);
+      LoggerService().log({
+        level: LogEntryLevel.Error,
+        area: 'Templates.usePanelTabs',
```

Comment:
Please use the area as Templates.createTab .... this clearly indicates this was during when create was happening... usePanelTabs is the file name and wont clearly identify what it was doing while checking telemetry